apps/react-native/babel.config.js

```diff
@@ -0,0 +1,33 @@
+module.exports = function(api) {
+  api.cache(true);
+
+  const presets = [
+    "module:metro-react-native-babel-preset",
+    [
+      "@babel/preset-env", {
+        "targets": {
+          "esmodules": true,
+          "node":true
+        },
+        //TODO: get a ton of errors in the console during yarn start without this
+        "loose": true
+      }
+    ]
+  ];
+
+  const plugins = [
+    [
+      '@brandingbrand/kernel-babel-plugin-dev', {
+        'project': 'react-native', // project root folder name
+        'screensDirectory': 'src/screens', // project screens to wrap with default dev menu on compile
+        'projectDevMenusDirectory': '__dev__', // (optional) custom project dev menus (relative to screensDirectory)
```

Comment:
let's rename to `devScreensPath` or `devScreens`

Reply:
I chose not to name it screens because they're not technically screens, just components that get rendered in the dev menu modal.